runtime/trappist/src/lib.rs

```diff
@@ -350,6 +353,7 @@ impl pallet_collator_selection::Config for Runtime {
 impl pallet_sudo::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type RuntimeCall = RuntimeCall;
+	type WeightInfo = ();
```

Comment:
Perhaps not worth the effort, but maybe this pallet should be benchmarked as best practice?

Reply:
this can be included in the next release.